hrqb/tasks/sql/employees.sql

```diff
@@ -3,8 +3,31 @@ Query for Employee data.
 
 CHANGELOG
     - 2024-05-13 Query created and added
+    - 2024-07-09 Added CTEs to gather last appointment transaction, to get termination
+        or retirement reason
 */
 
+with ordered_appt_txns as (
```

Comment:
Optional: Even though `tx` is used in the table, I think `transaction` would be much clearer to future devs

Reply:
I hesitate to use "transaction" a) because it's longer and b) because it really isn't used anywhere in the table where "TX" is widely used.   But see other comment, good suggestion on normalizing usage!